filter_temporal.json

```diff
@@ -33,19 +33,14 @@
                         {
                             "type": "string",
                             "format": "date-time",
-                            "subtype": "date-time"
+                            "subtype": "date-time",
+                            "description": "Date and time with a time zone."
```

Comment:
as mentioned above: time zone is optional I think and UTC by default?

Reply:
No, tz is required.